src/solution_check.jl

```diff
@@ -0,0 +1,105 @@
+function check_tw_bounds(tws::Vector{Range}, target_time::Float64)
+    
+    for tw in tws
+        if target_time <= tw.ub && target_time >= tw.lb
+            return true
+        end
+    end
+    return false
+
+end
```

Comment:
The function has nothing specific about time windows. therefore generic names would be more appropriate